esmvaltool/diag_scripts/miles/miles_regimes.R

```diff
@@ -0,0 +1,107 @@
+# #############################################################################
```

Comment:
Is this file still used? It looks outdated.

Reply:
It is actually for a planned (not yet implemented) future extension, but I removed it for now in https://github.com/ESMValGroup/ESMValTool/pull/735/commits/52d766c7462c31aeec5bda4f0b5d8e891bd2cfe2